plugins/services/src/js/components/ServiceConfigDisplay.js

```diff
@@ -9,29 +9,32 @@ import ServiceConfigEnvironmentVariablesSectionDisplay from './ServiceConfigEnvi
 import ServiceConfigLabelsSectionDisplay from './ServiceConfigLabelsSectionDisplay';
 import ServiceConfigHealthChecksSectionDisplay from './ServiceConfigHealthChecksSectionDisplay';
 
+const MOUNT_TYPE = 'CreateService:ServiceConfigDisplay';
+
 // To pad priorities, so we can add components in between
 const PRIORITIES_PAD_NUMBER = 100;
 
-const SERVICE_CONFIG_DISPLAY_LIST = [
+// Register default config display components (this needs to only happen once)
+[
   ServiceConfigGeneralSectionDisplay,
   ServiceConfigNetworkingSectionDisplay,
   ServiceConfigStorageSectionDisplay,
   ServiceConfigEnvironmentVariablesSectionDisplay,
   ServiceConfigLabelsSectionDisplay,
   ServiceConfigHealthChecksSectionDisplay
-].reverse(); // MountService sorts by priority but DESC ¯\_(ツ)_/¯
-
-class ServiceConfigDisplay extends React.Component {
-  componentWillMount() {
-    SERVICE_CONFIG_DISPLAY_LIST.forEach((component, index) => {
+]
+    .forEach((component, index, componentArray) => {
+      // Define  component priorities based on index (highest prio goes first)
+      // and pad priorities, so that we can add components in between.
+      let priority = (componentArray.length - index) * PRIORITIES_PAD_NUMBER;
```

Comment:
I think this will break the order I'm relying on in https://github.com/mesosphere/dcos-ui-plugins-private/pull/462/files#diff-f1e16cc6041efa19213bfb8863789802R71 I'm ok changing it in the plugin.